.github/workflows/ghcr.yml

```diff
@@ -35,6 +34,10 @@ jobs:
             type=sha
             type=ref,event=tag
 
+      - name: Set PNPM_VERSION
+        run: |
+          export PNPM_VERSION=$(cat .pnpm-version)
```

Comment:
I'm a bit surprised that this works. GitHub Actions [docs recommend `echo`-ing to `$GITHUB_ENV`](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable).

Reply:
Ah, I will do that!

@obulat that is probably why it failed :slightly_smiling_face: